src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/TandemRepeat.java

```diff
@@ -58,4 +58,10 @@
                 GATKVCFConstants.REPEATS_PER_ALLELE_KEY);
     }
 
+    private static byte[] getRefBases(final ReferenceContext ref, final VariantContext vc) {
```

Comment:
I just want to confirm that there is a test that breaks given the bug that this branch fixes.  In other words, a test that would fail without this fix.  Unless you do not think that is useful.